pkgs/development/tools/vulkan-validation-layers/default.nix

```diff
@@ -50,14 +67,13 @@ stdenv.mkDerivation rec {
     libXrandr
     libffi
     libxcb
-    spirv-tools
     vulkan-headers
     wayland
   ];
 
   cmakeFlags = [
     "-DGLSLANG_INSTALL_DIR=${glslang}"
-    "-DSPIRV_HEADERS_INSTALL_DIR=${spirv-headers}"
+    "-DSPIRV_HEADERS_INSTALL_DIR=${spirv-headers'}"
```

Comment:
Typo?

Reply:
No, intentional to use the override above.

Reply:
Oh :D sorry mb

Reply:
I don't know if there's precedent for using `'` this way in nixpkgs, but I think it makes for pretty poor readability.